spec.bs

```diff
@@ -579,11 +572,20 @@ Moreover, each {{SharedStorageWorklet}}'s [=global scopes|list of global scopes=
         1. If |filteringIdMaxBytes| is not the [=default filtering ID max bytes=] or
             |contextId| is not null, return a new {{DOMException}} with name
             "`DataError`".
+    1. Let |maxContributions| be null.
```

Comment:
We may want to define this a PR into the other PR so it doesn't include both changes

Reply:
Good point, forgot to call that out. I've added `DO_NOT_MERGE` to the PR title for the time being.